templates/default/config/siteConfig.js

```diff
@@ -21,23 +23,28 @@ const defaultConfig = {
   ],
   // optional additional nextSeo content set on each page
   // see https://github.com/garmeeh/next-seo
-//  nextSeo: {
-//    openGraph: {
-//      images: [
-//        {
-//          url: 'https://image.url/...',
-//          alt: '',
-//          width: 1200,
-//          height: 627,
-//          type: 'image/png',
-//        }
-//      ]
-//    }
-//  }
+  nextSeo: {
```

Comment:
I'm not sure about this approach - i think we want to simply reproduce nextSeo stuff into e.g. userConfig rather than try to simplify it. This would means docs page for this would show what would be a template for what to copy into `config.js` and explain where it comes from.

Why? Because a) users can't access full nextSeo stuff b) we have to maintain the interface between our config and nextSeo.

Reply:
moved seo defaults to user config now and no seo in siteConfig.